src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2969,6 +2970,9 @@ static void _styleCopy(SvgStyleProperty* to, const SvgStyleProperty* from)
     if (from->flags & SvgStyleFlags::PaintOrder) {
         to->paintOrder = from->paintOrder;
     }
+    if (from->flags & SvgStyleFlags::Display) {
+        to->display = from->display;
+    }
```

Comment:
it also needs to be applied to _copyStyle() in tvgSvgCssStyle.cpp.

Reply:
thx!